src/main/java/MealDemo/service/MealPlanningServiceImpl.java

```diff
@@ -0,0 +1,96 @@
+package MealDemo.service;
+
+import MealDemo.dao.FrequencyRepository;
+import MealDemo.dao.MealRepository;
+import MealDemo.entity.Frequency;
+import MealDemo.entity.Meal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MealPlanningServiceImpl implements MealPlanningService {
+
+    // =========== REPOSITORIES ===========
+    private MealRepository mealRepository;
+    private FrequencyRepository frequencyRepository;
+
+
+    // =========== CONSTRUCTOR ===========
+    @Autowired
+    public MealPlanningServiceImpl(MealRepository mealRepository, FrequencyRepository frequencyRepository) {
+        this.mealRepository = mealRepository;
+        this.frequencyRepository = frequencyRepository;
+    }
+
+
+    // =========== IMPLEMENTATIONS ===========
+
+    // GET MEALS & SEQUENCES
+    @Override
+    public List<Meal> getMeals() {
+        return mealRepository.findAll();
+    }
+
+    @Override
+    public List<Frequency> getFrequencies() {
+        return frequencyRepository.findAll();
+    }
+
+    // FIND BY ID
```

Comment:
This is a common thing juniors love to do. If you not using the comment to show sectioning like you doing with:

```
// =========== IMPLEMENTATIONS ===========
```

Then the comment might be redudent and this one is 100% noise

```
// FIND BY ID
public Optional<Meal> findMealById(int mealId)
```

The method name already tells me. This comment would only be useful if the method is doing something odd or has a side effect like:

```
// This find actually increases a count on the DB as a side effect
```

That's a useful comment because it's informing the reader of something "unexpected"

Reply:
> This is a common thing juniors love to do. If you not using the comment to show sectioning like you doing with 
It's a thing then. Glad Im not the only one. 

Will keep it to more informative comments in the future